sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/cryptography/KeyEncryptionKeyClientTestBase.java

```diff
@@ -71,7 +71,10 @@ HttpPipeline getHttpPipeline(HttpClient httpClient, CryptographyServiceVersion s
         }
         HttpPolicyProviders.addAfterRetryPolicies(policies);
         policies.add(new HttpLoggingPolicy(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS)));
-        policies.add(interceptorManager.getRecordPolicy());
+
+        if (!interceptorManager.isPlaybackMode()) {
```

Comment:
We only use record policy in RECORD mode. Should exclude LIVE test mode here as well. Same as below

Reply:
@sima-zhu We probably should modify the other `TestBase` classes in our repository if that's the case. They currently use a record policy when **not** in playback mode.

Reply:
Could you point out the place mentioned use RecordPolicy when not playback.
We introduce LIVE mode slightly later than the other modes, so it might not up-to-date.

Reply:
Besides the two classes included in this PR, in KeyVault you can find this in the following classes:
- [CertificateClientTestBase](https://github.com/Azure/azure-sdk-for-java/blob/2e2b719ddd9106c0b4a7c046a5ac8513612ff0b1/sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java#L106)
- [CryptographyClientTestBase](https://github.com/Azure/azure-sdk-for-java/blob/2e2b719ddd9106c0b4a7c046a5ac8513612ff0b1/sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientTestBase.java#L83)
- [KeyClientTestBase](https://github.com/Azure/azure-sdk-for-java/blob/2e2b719ddd9106c0b4a7c046a5ac8513612ff0b1/sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTestBase.java#L92)